tests/checkup_test.go

```diff
@@ -283,19 +283,27 @@ func newRoleBinding(name, serviceAccountName, roleName string) *rbacv1.RoleBindi
 }
 
 func newConfigMap() *corev1.ConfigMap {
+	testConfig := map[string]string{
```

Comment:
can this commit be in another PR? It's not a refactor, it's more a fix, no?

Reply:
Please see PR #181.